public class App {
        static int getGreeting(int b) {
            b=6;
            return b;
        }
    public static void main(String[] args) throws Exception {
        int a=5;

        System.out.println(getGreeting(a));
        System.out.println(a);

        Math.pow()

        //Test t=new Test();
        //System.out.println(t.x);
        // int a=-1;
        // System.out.println(a<<2);
        // System.out.println(a>>2);
        // System.out.println(a>>>4);
    }
}
